src/main/java/stream/flarebot/flarebot/commands/general/TagsCommand.java

```diff
@@ -0,0 +1,88 @@
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.TextChannel;
+import net.dv8tion.jda.core.entities.User;
+import org.apache.commons.lang3.StringUtils;
+import stream.flarebot.flarebot.commands.Command;
+import stream.flarebot.flarebot.commands.CommandType;
+import stream.flarebot.flarebot.objects.GuildWrapper;
+import stream.flarebot.flarebot.util.MessageUtils;
+
+import java.awt.Color;
+
+public class TagsCommand implements Command {
+
+    @Override
+    public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Message message, String[] args, Member member) {
+        if (args.length == 0) {
+            if (guild.getTags().isEmpty()) {
+                MessageUtils.sendErrorMessage("There are currently no tags for this guild!\n" +
+                        "Add a tag via `{%}tags add <tag_name> <tag_message>`.", channel);
+                return;
+            }
+            channel.sendMessage(MessageUtils.getEmbed(sender).setTitle("Guild Tags", null).setColor(Color.cyan)
+                    .setDescription("```\n" + StringUtils.join(guild.getTags().keySet(), ", ") + "\n```")
+                    .build()).queue();
+        } else if (args.length == 1) {
+            if (guild.getTags().containsKey(args[0].toLowerCase()))
+                channel.sendMessage(guild.getTags().get(args[0].toLowerCase())).queue();
+            else
+                MessageUtils.sendErrorMessage("This tag doesn't exist!", channel);
+        } else if (args.length == 2) {
+            if (args[0].equalsIgnoreCase("add")) {
+                MessageUtils.sendErrorMessage("This seems to be invalid ¯\\_(ツ)_/¯\n"
+                        + "Usage: `{%}tags add " + args[1] + " <tag_message>`", channel);
+            } else if (args[0].equalsIgnoreCase("remove")) {
+                if (guild.getTags().containsKey(args[1].toLowerCase())) {
+                    guild.getTags().remove(args[1]);
+                    MessageUtils.sendSuccessMessage("You successfully removed the tag `" + args[1] + "`!",
+                            channel, sender);
+                } else
+                    MessageUtils.sendErrorMessage("This tag doesn't exist!", channel);
+            } else
+                MessageUtils.sendErrorMessage("That is an invalid argument!", channel);
+        } else {
+            if (args[0].equalsIgnoreCase("add")) {
+                if (guild.getTags().containsKey(args[1].toLowerCase())) {
+                    MessageUtils.sendErrorMessage("This tag already exists!", channel);
+                    return;
+                }
+
+                guild.getTags().put(args[1].toLowerCase(), MessageUtils.getMessage(args, 2));
+                MessageUtils.sendSuccessMessage("You successfully added the tag `" + args[1] + "`!", channel,
+                        sender);
+            } else
+                MessageUtils.sendErrorMessage("That is an invalid argument!", channel);
```

Comment:
Use sendusage

Reply:
You love the sendUsage :p honestly I don't think it should be abused that much until we write the better sendUsage we planned